src/features/landing/components/Footer/FooterCopyright/FooterCopyright.tsx

```diff
@@ -0,0 +1,28 @@
+import { Typography } from '@mui/material';
```

Comment:
for what reason was created this component? it looks like the component is made for the sake of the component